packages/document/main-doc/docs/en/guides/concept/builder.mdx

```diff
@@ -4,31 +4,31 @@ sidebar_position: 2
 
-Modern.js Builder is one of the core components of Modern.js. It is A build engine for web development. and can be used independently of Modern.js. Modern.js Builder supports multiple bundlers such as webpack and Rspack, and it uses webpack by default.
+Modern.js Builder is one of the core components of the Modern.js system. It is a build engine for web development scenarios that can be used independently of Modern.js. Modern.js Builder supports multiple packaging tools such as webpack and Rspack, and by default uses the most mature webpack for packaging.
 
 ## Build Architecture
 
-From the perspective of building, Modern.js is divided into three layers, from top to bottom:
+From a building perspective, Modern.js is divided into three layers, from top to bottom:
 
-- Upper-layer framework: Modern.js.
-- Universal build engine: Modern.js Builder.
-- Low-level bundlers: webpack and rspack.
+- Upper-level development framework: Modern.js.
+- Common build engine: Modern.js Builder.
+- Lower-level packaging tool: webpack and Rspack.
 
 <img src="https://lf3-static.bytednsdoc.com/obj/eden-cn/zq-uylkvT/ljhwZthlaukjlkulzlp/builder-layers-1117.png" style={{ width: '100%', maxWidth: '540px' }} />
 
 ## Builder Documentation
 
-Since Modern.js Builder is a module that can be used independently, we provide a separate document for it, the site address is: [modernjs.dev/builder](https://modernjs.dev/builder).
+Since Modern.js Builder is a module that can be used independently, we have provided separate documentation for it, which can be found at [modernjs.dev/builder](https://modernjs.dev/builder).
 
-In this document, you can learn about the detailed introduction of Modern.js Builder, and you can also find a complete usage guide for each building features. When you encounter building issues, it is recommended that you first read the Modern.js Builder documentation to solve them.
+In this documentation, you can learn about the detailed introduction of Modern.js Builder, and you can also find complete usage guides for various building capabilities. When you have building needs or problems, it is recommended that you read the Modern.js Builder documentation first to solve them.
 
-## Builder Config
+## Builder Configuration
 
-The config of Modern.js is inherited from Modern.js Builder, so you can use all build configs provided by Modern.js Builder in Modern.js.
+Modern.js's configuration inherits from Modern.js Builder, so you can use all the build configurations provided by Modern.js Builder in Modern.js.
 
-Take the `html.title` config of Modern.js Builder as an example, you can directly use this config in the `modern.config.ts` file, and it will be automatically passed to Modern.js Builder.
+Taking the `html.title` configuration item of Modern.js Builder as an example, you can directly use this configuration item in the `modern.config.ts` file, and it will be automatically passed to Modern.js Builder.
```

Comment:
I find that `config option` is more commonly used by english documentation sites than `configuration item`

```suggestion
Taking the `html.title` config option of Modern.js Builder as an example, you can directly use this option in the `modern.config.ts` file, and it will be automatically passed to Modern.js Builder.
```